source/includes/gho/_api.md

```diff
@@ -0,0 +1,302 @@
+start_date | [DateFilter](#datefilter) | 
+date_of_birth | [DateFilter](#datefilter) | 
+with_custom_field_value | [WithCustomFieldValue](#withcustomfieldvalue) | 
+# Mutations
+## updateEmployeeProfile \([Employee](#employee)\)
+Update an employee's profile
+
+Argument | Type | Description
+--------- | ----------- | -----------
+id | [ID](#id) | 
+employee_updates | [UpdateEmployee](#updateemployee) | 
+# Types
+## CustomField
+    Represents a single CustomField record for your company.  CustomFields can be stored and displayed in a variety of    ways.  The types are described via the [CustomFieldTypes](#customfieldtypes) enum.  
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+field_type | [CustomFieldTypes](#customfieldtypes) | The field type determines how users input and view the data for this field.
+id | [ID](#id) | 
+name | [String](#string) | The name of this custom field as users would see it inside Greenhouse Onboarding.
+permanent_field_id | [String](#string) | A unique identifier for this CustomField.
+updated_at | [DateTime](#datetime) | 
+
+## CustomFieldValue
+A Custom Field Value Record
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+custom_field | [CustomField](#customfield) | 
+id | [ID](#id) | 
+updated_at | [DateTime](#datetime) | 
+value | [Value](#value) |     A different type of value will be stored based upon the field type of the [CustomField](#customfield).  Some types    have the data stored as a nested object.  Note that the type is a scalar named [Value](#value).  Even though    it appears to be an object, you are not able to use GraphQL to determine its shape.  
+value_updated_at | [DateTime](#datetime) | The time of the most recent update to this field.
+
+## Department
+    Represents a single department in your company.  Employees may belong to zero or one department. Departments are    used in a variety of ways in Greenhouse Onboarding, including permissions and onboarding plans.  
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+description | [String](#string) | 
+id | [ID](#id) | 
+name | [String](#string) | 
+updated_at | [DateTime](#datetime) | 
+
+## Document
+Represents a single document attached to an [Employee](#employee).
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+file | [File](#file) | Contains the file payload.
+id | [ID](#id) | 
+updated_at | [DateTime](#datetime) | 
+
+## Employee
+A single Employee that works for your company.
+
+Field | Type | Description
+--------- | ----------- | -----------
+about | [String](#string) |     A brief description of the employee.  This information is displayed on both the employee's profile and is also    featured prominently in the Welcome Experience for any new hires that report to them.  
```

Comment:
I'm not sure what this sentence means because I don't know who "them" is referring to at the end of the sentence. Are you trying to say the information is displayed to the employee's boss or task owners? "This information is displayed on the employee's profile and featured prominently on the Welcome Experience page for task owners."?